docs/conf/prague/2021/job-fair.rst

```diff
@@ -14,3 +14,26 @@ Schedule
 --------
 
 Scheduling information is available on our :doc:`/conf/{{shortcode}}/{{year}}/schedule` page.
+
+
+Tips for Attendees
+------------------
+
+If you're attending the Virtual Job Fair, make sure you're signed in to Hopin.
+
+* Select Expo in the left-hand pane.
+* Select the company of your choice.
+* You now have the following choices:
+
+  * Chat with attendees from the company under the Chat tab.
+  * Select the button to "Share Audio and Video". You can then chat "Face to Face" with company representatives.
+
+Tips for Sponsors
+-----------------
+
+If you want to maximize your contacts with potential new employees, make sure to:
+
+* Set up moderator privileges. To minimize delays, set this up **before the conference**. If needed, you can also request access from the designated Job Fair volunteer from the Write the Docs conference.
```

Comment:
I don't think they'll have any idea what moderator privileges setup means. How I've been doing it behind the scenes is adding everyone that seems to work for that sponsor as a moderator.

```suggestion
* Make sure your booth staff is registered in the online platform before the conference starts. All ticket holders will receive details about this a few days before the conference. These staff will be set up as moderators for the booth If needed, you can also request additional moderators to be set up from the designated Job Fair volunteer from the Write the Docs conference.
```

Reply:
I appreciate where you're going, I hope this makes your intent more clear, @mxsasha .

```suggestion
* Make sure your booth staff is registered in the online platform before the conference starts. All sponsor ticket holders will receive details about this a few days before the conference. These staff will be set up as moderators for your booth. As needed, you can also request additional moderators from the designated conference Job Fair volunteer.
* ```